src/kron.jl

```diff
@@ -0,0 +1,56 @@
+export opKron
+
+function opKron(A :: AbstractMatrix,
```

Comment:
You don't need access to the elements of either operator because of the identity kron(B', A) * vec(X) = vec(AXB). In other words, if C = kron(B', A), and x is a vector of appropriate size, then Cx = AXB, where X is the vector x reshaped as a matrix. Maybe your version is faster if one of the operands is an explicit matrix because with what I suggest, we then need full(AXB), which should then be reshaped as a vector.

Reply:
I've tested this and the `full` way is faster, so I'll change everything to it.